TTS/engine_wrapper.py

```diff
@@ -70,7 +71,16 @@ def run(self) -> Tuple[int, int]:
             if not self.tts_module.max_chars:
                 self.call_tts(f"{idx}", comment["comment_body"])
             else:
-                self.split_post(comment["comment_body"], idx)
+                # THESE TRY EXCEPTS SOMEHOW SOLVES FFMPEG ERROR #
```

Comment:
this would cause audio de-syncing I believe.

I fixed the actual cause of this error on https://github.com/elebumm/RedditVideoMakerBot/tree/feat/background-audio